taprpc/priceoraclerpc/price_oracle.proto

```diff
@@ -32,13 +63,21 @@ enum TransactionType {
 // NOTE: That if the payment asset is BTC, the rate tick will be given as
 // milli-satoshi per asset unit.
 message RateTick {
-    // rate is the exchange rate between the subject asset and the payment
-    // asset.
-    uint64 rate = 1;
+    // subjectAssetRate is the subject asset to BTC conversion rate
```

Comment:
The comment above about the tick is now outdated.

Reply:
Also I think we should find a place in the proto comments to insert the formulas we have for conversion. Will serve as a guide for those implementing the new oracle version.

Reply:
A lot of `price_oracle.proto` is out of date. The rate tick terminology for example. I will be sure to change all that before I consider this work complete. But for this PR I hope to just focus on getting the fixed-point type into the right fields.

I'll be sure to add a lot to the proto comments :+1: